pyro/distributions/multivariate_cauchy.py

```diff
@@ -0,0 +1,39 @@
+import torch
+from torch.distributions import constraints
+from pyro.distributions.transforms import LowerCholeskyAffine
+from pyro.distributions.torch import TransformedDistribution, Cauchy
+
+
+class MultivariateCauchy(TransformedDistribution):
+    r"""
+    Creates a multivariate Cauchy distribution parameterized by
+    :attr:`loc` and :attr:`scale_tril` where::
+
+        X ~ Cauchy(loc, 1.0)
+        Y = scale_tril @ X ~ MultivariateCauchy(loc, scale_tril)
+
+    :param torch.tensor loc: the location parameter of the distribution, a D-dimensional vector.
+    :param torch.tensor scale_tril: the scale parameter of the distribution, a D x D lower-triangular matrix.
+    """
+    arg_constraints = {'loc': constraints.real, 'scale_tril': constraints.lower_triangular}
+    support = constraints.real
+    has_rsample = True
+
+    def __init__(self, loc, scale_tril, validate_args=None):
+        base_dist = Cauchy(loc, 1.0).to_event(1)
+        transforms = [LowerCholeskyAffine(torch.zeros_like(loc), scale_tril)]
+        self.loc = loc
```

Comment:
I don't think this will work correctly with `.expand`. Maybe that's why those `@property`s were there before. Maybe we should
```py
@property
def loc(self):
    return self.base_dist.loc

@property
def scale_tril(self):
    return self.transforms[0].scle_tril
```

Reply:
Sorry I missed this refactoring. This will throw an error currently because `expanded_dist.loc` will be missing.